docs/open-api.yaml

```diff
@@ -33,13 +97,13 @@ paths:
         type: "AWS_PROXY"
       tags:
         - NfrVerifiedFundingSources
-      summary: Get NFR funding source by identifier
-      description: Returns a funding source by identifier
-      operationId: NfrGetFundingSource
+      summary: Get NFR funding by identifier
+      description: Returns a funding  by identifier
+      operationId: NfrGetFunding
       parameters:
         - name: identifier
           in: path
-          description: The funding source identifier
```

Comment:
Why source is wrong? 
Funding is combination of code and id? NRE:1234
Or just funding source identifier? Cristin funding id 1234? In the example below  we use id not the funding string or I am mistaken?
I rely on this table: https://api.cristin-test.uio.no/v2/doc/index.html#GETfundings (might not link you straight)

Reply:
Answered in slack